model/src/main/scala/za/co/absa/atum/model/dto/CheckpointDTO.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model.dto
+
+import za.co.absa.atum.model._
+
+import java.time.ZonedDateTime
+import java.util.UUID
+
+case class CheckpointDTO(
+  id: UUID,
+  name: String,
+  author: String,
+  measurementsProvidedManually: Boolean = false,
```

Comment:
Feels like a mouth-full sorry. 😊 
```suggestion
  measuredByAtum: Boolean = true,
```
(Although I would prefer the default to be `false` 🤔 )

Reply:
how about `measuredByAtumAgent` ? 

Okay, `false` is good, agreed - at least we will explicitly set this to `true` in the Agent and whoever will send measurements that were computed by some other process other than the agent, might not want to care about this parameter